missions/EIT_Missions.txt

```diff
@@ -1052,25 +1460,51 @@ eientei_missions_middle_screen_slot = {
 		position = 9
 		provinces_to_highlight = {
 			OR = {
-				area = Scarlett_domain
-				area = Scarlet_island
-				area = Scarlet_mountain
+				area = scarlett_domain
+				area = scarlet_island
+				area = scarlet_mountain
+				area = nightvault_area 
+				area = filamentous_spire_area 
+				area = chateau_flandre_area 
+				area = starfall_area
+				area = scarlet_mountain
+				area = scarlett_domain
+				area = scarletta_area 
+				area = draculas_haunt_area
+				area = sanguine_citadel_area 
+				area = scarlet_gate_area 
+				area = scarlet_kitchen_area 
+				area = scarlet_island
+				area = izayoi_area
 			}
 			NOT = {
 				country_or_non_sovereign_subject_holds = ROOT
 			}
 		}
 		trigger = {
 			calc_true_if = {
-				all_province = {
+				all_owned_province = {
```

Comment:
Changing from `all_province` to `all_owned_province` makes subject provinces no longer count
```suggestion
				all_province = {
```